wiki/Performance_Points/en.md

```diff
@@ -1,159 +1,116 @@
+*Aim* is a core value that considers how difficult it is to consistently hit consecutive notes in a beatmap.
 
-Aim is not considered in gamemodes like _osu!taiko_ and _osu!mania_.
+Elements like [approach rate](/wiki/Beatmapping/Approach_rate) and certain [mods](/wiki/Game_Modifiers) (namely [Flashlight](/wiki/Game_Modifiers#flashlight), [Hidden](/wiki/Game_Modifiers#hidden) and [Hard Rock](/wiki/Game_Modifiers#hard-rock)) make navigating the cursor quickly and accurately significantly more difficult, and thus influence the amount of pp a score gives.
 
-### What is speed?
+In the case of [osu!standard](/wiki/Game_Modes/osu!), beatmaps with very large [jumps](/wiki/Beatmaps/Pattern/Jump) are considered to be "high aim" beatmaps, and are thus often given very high pp scores. Likewise, beatmaps with more hyperdashing in [osu!catch](/wiki/Game_Modes/osu!catch) will be considered similarly. Aim is not considered in gamemodes like [osu!taiko](/wiki/Game_Modes/osu!taiko) and [osu!mania](/wiki/Game_Modes/osu!mania).
 
-**Speed is considered as the rate at which a beatmap presents elements for play.**
+### Speed
 
-Maps with high numbers of hit objects in a short period of time are considered to have very high speed values.
+<!-- A "see also" or some other sort of hatnote for the--potential--pp farm stub -->
 
-Mods like DoubleTime and HalfTime significantly affect the speed of a beatmap as considered by the performance points algorithm.
+*Speed* is a core value that considers the rate at which a beatmap presents elements for a play.
 
-### What is accuracy?
+Beatmaps with high numbers of hit objects in a short period of time are considered to have very high speed values. In this specific aspect, the faster a beatmap's speed is, the more difficult said beatmap is, therefore granting larger gains of pp.
 
-**Accuracy is considered as your individual performance and consistency in hitting objects within their designated timeframe.**
+As a result of this, mods like [Double Time](/wiki/Game_Modifiers/Half-Time) and [Half Time](/wiki/Game_Modifiers#Half-Time) significantly affect the speed of a beatmap considered by the performance points algorithm. Likewise, these mods also significantly affect pp gains when used.
 
-Highly accurate scores are considered to be very skilful by the performance points algorithm and will award very large scores compared to scores that are not accurate.
+### Accuracy
 
-A score set with 80% accuracy is sometimes worth 2/3 of a score set with 95% accuracy, for example.
+*See also: [Accuracy](/wiki/Accuracy)*
 
-Mods like Hidden, Hard Rock and Flashlight are considered to significantly alter how difficult it is to attain an accurate score on a beatmap.
+*Accuracy* is a percent measure of a player's ability to hit [hit objects](/wiki/Beatmaps/Hit_object) on-time; in regards to the pp algorithm, it is also a core value that is used to evaluate a player's individual performance on a beatmap.**
```

Comment:
What is that that `**` at the end doing?

Reply:
Prob an accidental copy-paste; my bad